sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1003,19 +1014,19 @@ public String getId() {
         String itemId,
         PartitionKey partitionKey,
         CosmosPatchOperations cosmosPatchOperations,
-        CosmosItemRequestOptions options,
+        CosmosPatchItemRequestOptions options,
```

Comment:
I know that this is not introduced in this PR. So we don't have to find a solution to it in this PR. 
But we need to pay attention what we should do.

We have a method here with 5 args which IMO in Java is not as great. We have tried to limit the number of args in the public surface area for better usability.

@FabianMeiswinkel @kushagraThapar @milismsft what's your thought on a method with 5 args here? thought?

Reply:
As discussed offline - I don't have a proposal for a better way (that is consistent with the approach for other point operations). itemId, pk and item Type are required. The only alternative I see is to push id, pk and itemType into CosmosPatchOperations as required parameters for the static create factory method.
But IMO this is more confusing because it could create the impression that path operations can spread across documents/items - I prefer the current mode which allows factoring the patchOperations and reuse it for different documents.